src/main/integrations/additional-context.ts

```diff
@@ -44,21 +44,26 @@ export class AdditionalContext implements Integration {
     addGlobalEventProcessor(async (event: Event) => this._addAdditionalContext(event));
 
     // Some metrics are only available after app ready so we lazily load them
-    void whenAppReady.then(() => {
-      const { language, screen } = this._options;
+    whenAppReady.then(
```

Comment:
Just fyi, when you're 100% certain that a promise cannot reject it is also fine to just ignore the lint rule!